docs/customizations/operation-badges.md

```
@@ -21,22 +23,20 @@ outline: false
-</script>
+<template #example>
 
-<OAOperation :spec="spec" operationId="getAllArtists" :isDark="isDark" />
-```
+<!--@include: ./parts/operation-badges-example.md-->
+
+</template>
+
+</ExampleBlock>
+
+# Custom Prefix
```

Markdownlint

Multiple top-level headings in the same document

(MD025, single-title, single-h1)